calculator.js

```diff
@@ -104,23 +104,27 @@ negativeValueButton.addEventListener("click", function() {
 
 addingButton.addEventListener("click", function() {
     operator = "+";
-    lastValue = result.textContent;
     lastButtonClicked = "+";
+    lastValue = result.textContent;
+    result.textContent = lastValue + lastButtonClicked;
```

Comment:
I don't know why you put this line again `result.textContent = lastValue + lastButtonClicked;` despite a few days ago I told you to remove it.

Reply:
I putted this time after adding this line :
  lastButtonClicked = "+";

and i putted to use the lastButtonClicked variable in this handler

Reply:
you're using the `lastButtonClicked ` in handler

Here you're setting the value of the  `lastButtonClicked ` to "+" ; so you're using it 
`  lastButtonClicked = "+";`